test/e2e-test/pages/overview-page/overview-page.ts

```diff
@@ -50,6 +50,13 @@ module.exports = {
       I.seeInSource(`${i18n.pages.overviewPage.doThisNext.clarifyingQuestions.respondByTextAskForMoreTime}`);
     });
 
+    Then(/^I should see the 'do this next section' for 'Your appeal cannot continue'$/, () => {
```

Comment:
If we can let's avoid adding more steps and keep it as generic as possible, I think there's a step you could use to check subheadings

Reply:
agreed, removed